amethyst_config/src/lib.rs

```diff
@@ -115,7 +117,7 @@ where
 {
     fn load<P: AsRef<Path>>(path: P) -> Self {
         Self::load_no_fallback(path.as_ref()).unwrap_or_else(|e| {
-            println!("1: Failed to load config: {}", e);
+            error!("Failed to load config: {}", e);
```

Comment:
On Linux at least, this error just says `No such file or directory`. It would be very helpful to print the path here as well.